x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/index.tsx

```diff
@@ -0,0 +1,415 @@
+ * 2.0.
+ */
+import { EuiFlexGroup, EuiFlexItem, EuiHideFor } from '@elastic/eui';
+import React, { useMemo, useCallback, useState, useRef } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { generateFilters } from '@kbn/data-plugin/public';
+import type { DataViewField } from '@kbn/data-plugin/common';
+import { DataView } from '@kbn/data-views-plugin/public';
+import type { SortOrder } from '@kbn/saved-search-plugin/public';
+import type { DataLoadingState } from '@kbn/unified-data-table';
+import { useColumns } from '@kbn/unified-data-table';
+import { popularizeField } from '@kbn/unified-data-table/src/utils/popularize_field';
+import type { DropType } from '@kbn/dom-drag-drop';
+import styled from 'styled-components';
+import { DragDrop, DropOverlayWrapper, useDragDropContext } from '@kbn/dom-drag-drop';
+import type {
+  UnifiedFieldListSidebarContainerApi,
+  UnifiedFieldListSidebarContainerProps,
+} from '@kbn/unified-field-list';
+import { UnifiedFieldListSidebarContainer } from '@kbn/unified-field-list';
+import type { EuiTheme } from '@kbn/react-kibana-context-styled';
+import type { CoreStart } from '@kbn/core-lifecycle-browser';
+import { EventDetailsWidthProvider } from '../../../../common/components/events_viewer/event_details_width_context';
+import type { ExpandedDetailTimeline } from '../../../../../common/types';
+import type { TimelineItem } from '../../../../../common/search_strategy';
+import { useKibana } from '../../../../common/lib/kibana';
+import { defaultHeaders } from '../body/column_headers/default_headers';
+import type {
+  ColumnHeaderOptions,
+  OnChangePage,
+  RowRenderer,
+  SortColumnTimeline,
+  ToggleDetailPanel,
+  TimelineTabs,
+} from '../../../../../common/types/timeline';
+import type { State, inputsModel } from '../../../../common/store';
+import { SourcererScopeName } from '../../../../common/store/sourcerer/model';
+import { useSourcererDataView } from '../../../../common/containers/sourcerer';
+import { getColumnHeader } from '../body/column_headers/helpers';
+import { timelineBodySelector } from '../body/selectors';
+import { StyledPageContentWrapper, StyledMainEuiPanel, StyledSplitFlexItem } from './styles';
+import { DRAG_DROP_FIELD } from './data_table/translations';
+import { TimelineResizableLayout } from './resizable_layout';
+import TimelineDataTable from './data_table';
+import { timelineDefaults } from '../../../store/defaults';
+import { timelineActions } from '../../../store';
+import { getFieldsListCreationOptions } from './get_fields_list_creation_options';
+
+const TimelineBodyContainer = styled.div.attrs(({ className = '' }) => ({
+  className: `${className}`,
+}))`
+  width: 100%;
+  height: 100%;
+`;
+
+const DataGridMemoized = React.memo(TimelineDataTable);
+
+const DROP_PROPS = {
+  value: {
+    id: 'dscDropZoneTable',
+    humanData: {
+      label: DRAG_DROP_FIELD,
+    },
+  },
+  order: [1, 0, 0, 0],
```

Comment:
What does this do. Worth adding a comment?

Reply:
Even I need to check this. Thanks.

Reply:
Okay it seems kbn-draggable needs these params as-is. Honestly I am not yet sure what this array exactly means.. I will try to find it out soon but let's try to move forward with this PR, i will add the description to this as I will have more info.

Reply:
gotta love that 😂 . Cool, thanks!